src/features/desktop.md

```diff
@@ -1,8 +1,8 @@
 # CheriABI desktop environment (experimental)
 
-As of the 22.12 release, the installer has gained the option to install a
+As of the 23.11 release, the installer has gained the option to install a
```

Comment:
Maybe this should just say “The installer has the option to ...” since it will be in two major releases now.